README.rst

```diff
@@ -153,12 +122,21 @@ Behavior
    causes nasty crashes by making it available through mass storage
    after the crash. A reset (the button) is needed after its fixed to
    get back into normal mode.
+-  RGB status LED
```

Comment:
... indicating state, and indicating errors through a sequence of colored flashes. [or something like that]